kibana-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -42,19 +42,19 @@ export function ReportDefinitionDetails(props) {
     setReportDefinitionDetails(e);
   };
 
-  const readableTimeRange = (data) => {
-    let timeRangeString = '';
-    if (
-      data['trigger']['trigger_params']['schedule']['interval']['unit'] ===
-      'DAYS'
-    ) {
-      timeRangeString +=
-        'Every ' +
-        data['trigger']['trigger_params']['schedule']['interval']['period'] +
-        ' days';
-    }
-    return timeRangeString;
-  };
+  // const readableTimeRange = (data) => {
```

Comment:
Can you remove these commented-out blocks?

Reply:
will do